hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -1687,3 +1679,74 @@ objects:
       targetRef:
         name: splunk-hec-token
         namespace: openshift-security
+
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    name: splunk-auth-region-global
+    namespace: splunk-forwarder-operator
+  spec:
+    clusterDeploymentSelector:
+      matchExpressions:
+        - key: api.openshift.com/managed
+          operator: In
+          values:
+            - "true"
+        - key: hive.openshift.io/cluster-region
+          operator: NotIn
+          values:
+            - "cn-north-1"
+            - "cn-northwest-1"
+        - key: api.openshift.com/fedramp
+          operator: NotIn
+          values:
+            - "true"
+        - key: ext-hypershift.openshift.io/regional-ocm
+          operator: NotIn
+          values:
+            - "true"
+    resourceApplyMode: Sync
+    secretMappings:
+    - sourceRef:
+        name: splunk-auth
+        namespace: splunk-forwarder-operator
+      targetRef:
+        name: splunk-auth
+        namespace: openshift-security
+
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    name: splunk-auth-region-ase1
+    namespace: splunk-forwarder-operator
+  spec:
+    clusterDeploymentSelector:
+      matchExpressions:
+        - key: api.openshift.com/managed
+          operator: In
+          values:
+            - "true"
+        - key: ext-hypershift.openshift.io/cluster-type
+          operator: In
+          values:
+            - "management-cluster"
+        - key: api.openshift.com/environment
+          operator: In
+          values:
+            - "production"
```

Comment:
Is there a form of this targeting the staging env where we can test the ingestions and query some samples?
I wonder if I could use it to test _OSD-26970_

Reply:
Do we have a staging environment for the Singapore regionalization?